# fix compilation error relating to undefined function (was it strlen?)

--- a/rosetta++/src/ObjexxFCL/string.functions.cc	2008-04-17 22:26:34.000000000 +0200
+++ b/rosetta++/src/ObjexxFCL/string.functions.cc	2008-12-05 22:39:40.000000000 +0100
@@ -17,6 +17,7 @@
 // C++ Standard Library Headers
 #include <algorithm>
 #include <cctype>
+#include <cstring>
 
 
 namespace ObjexxFCL {
